### PR TITLE
update docusaurus workflow

### PR DESCRIPTION
**Pull request by aabedraba**
This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: .github/workflows/docusaurus.yml
Comment (kylehodgetts) [inline .github/workflows/docusaurus.yml:27]: We might need to run this through some tests, somehow, before merging.
My concern is that you are `cd`ing into docs in the Install Dependencies step and then cd'ing in again here.
Does Github actions preserve current directory in between steps?

Comment (aabedraba) [reply .github/workflows/docusaurus.yml:27]: I ran the tests in my repo and it worked https://github.com/aabedraba/kusk-gateway-docs-2

**Review by kylehodgetts — APPROVED** [review]
Seen in action and it works. Ship it once the tests pass

**Pull request merged**
